frontend/src/component/feature/StrategyTypes/FlexibleStrategy/StickinessSelect/StickinessSelect.tsx

```diff
@@ -25,20 +24,27 @@ export const StickinessSelect = ({
     const { context } = useUnleashContext();
     const theme = useTheme();
 
-    const resolveStickinessOptions = () =>
-        builtInStickinessOptions.concat(
-            context
-                .filter(contextDefinition => contextDefinition.stickiness)
-                .filter(
-                    contextDefinition =>
-                        !builtInStickinessOptions.find(
-                            builtInStickinessOption =>
-                                builtInStickinessOption.key ===
-                                contextDefinition.name
-                        )
-                )
-                .map(c => ({ key: c.name, label: c.name }))
-        );
+    const resolveStickinessOptions = () => {
```

Comment:
We started adding more tests around new behaviors recently. This one should be easy to test using this approach: https://github.com/Unleash/unleash/blob/239e338daa430cca2ade8dbe2e61883e698b3b5e/frontend/src/component/changeRequest/ChangeRequestSidebar/EnvironmentChangeRequest/EnvironmentChangeRequestTitle.test.tsx

Reply:
Thanks, but I am cherry picking commits, not implementing anything here

Reply:
Yeah I noticed after leaving the comment :)

Reply:
Cherry picked the BE protection for empty string stickiness as well